ci-operator/step-registry/mco-day2/create-custom-mcp/mco-day2-create-custom-mcp-ref.metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"path": "mco-day2/create-custom-mcp/mco-day2-create-custom-mcp-ref.yaml",
+	"owners": {
+		"approvers": [
+			"yunjiang29",
+			"jianlinliu",
+			"jinyunma",
+			"Xia-Zhao-rh",
+			"kuiwang02",
+			"bandrade",
+			"jianzhangbjz"
+		],
+		"reviewers": [
+			"Xia-Zhao-rh",
+			"kuiwang02",
+			"bandrade",
+			"jianzhangbjz",
+			"sregidor",
+			"rioliu-rh"
+		]
+	}
+}
```

Comment:
warning at the end of file

Reply:
Done

Reply:
If I remove the new line at the end of the files, the metadata validation fails. Metadata files are processed by some automation tasks and they must be as they are when "make update" command finishes.

I have restored the original metadata files with the new line at the end of the files so that the metadata validation can pass.